WeightProducer/python/MCSampleValues.py

```diff
@@ -120,7 +116,7 @@ class MCSampleValuesHelper():
     __values_dict = {
         "TTJets" : {
             "CrossSection" : XSValues(
-                XS_13TeV=831.76, XSSource_13TeV="XSDB (NNLO)",
+                XS_13TeV=750.5, XSSource_13TeV="XSDB (unknown)",
```

Comment:
`831.76` is the inclusive NNLO XS from the PDG (https://pdg.lbl.gov/2019/reviews/rpp2019-rev-top-quark.pdf). It was most likely put in by hand. Where did you get 750.5? I suspect that's not correct and that likely this one shouldn't be changed.

Reply:
Yeah, these numbers coming from the XSDB do not have much context or justification. When possible, I will stick to the highest order calculated values from PDG, etc.